manifest/net.NepuShiro/ResoniteSlotInspectorCounter/info.json

```diff
@@ -25,6 +25,13 @@
 				"url": "https://github.com/NepuShiro/ResoniteSlotInspectorCounter/releases/download/1.6.0/ResoniteSlotInspectorCounter.dll",
 				"sha256": "a6d2523a41ace4306ada10fafc854f8678d7388236b82834203d8569ba72b2ab"
 			}]
+		},
+		"1.8.1": {
```

Comment:
Mod has `[assembly: AssemblyVersion("1.8.0.0")]` / `[assembly: AssemblyFileVersion("1.8.0")]` which doesn't match Version 1.8.1 set here and in mod